ui/package.json

```diff
@@ -60,6 +62,7 @@
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react": "^7.21.4",
     "eslint-plugin-react-hooks": "^4.1.2",
+    "lit": "^2.0.2",
```

Comment:
I'm not sure why it doesn't build without explicitly adding lit as a dependency. I would think it would get it transitively through @agoric/wallet-connection